src/Domain/POV/ConfigCreator.php

```diff
@@ -145,10 +150,11 @@ public function create(
         }
         $this->log('json decoded, extracting region from raster layers');
         $this->excludeLayersByTags($json['datamodel']['raster_layers'], $json['datamodel']['vector_layers']);
+        $this->fixMissingRasterLayerScales($json['datamodel']['raster_layers']);
         $this->extractRegionFromRasterLayers($region, $json['datamodel']['raster_layers'], $dir);
         if (false !== $bathymetryLayer = current(array_filter(
             $json['datamodel']['raster_layers'],
-            fn($x) => $x['name'] === 'Bathymetry'
+            fn($x) => empty(array_diff(['Bathymetry', 'ValueMap'], $x['tags']))
```

Comment:
matching Bathymetry layer now based on tags instead of the name (which was actually "short")